NOnion/Utility/MailboxUtil.fs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NOnion.Utility
+
+open System.Net.Sockets
+
+//FIXME: for some reason FSharpUtil is in NOnion namespace instead of NOnion.Utility
```

Comment:
@aarani let's fix it in a 2nd commit

Reply:
Fixed. open NOnion was still needed in that file for NOnionSocketException tho.